### ISSUE TITLE
2019.12 - cron.sh - MySQL Port not set to default

### ISSUE THREAD

**Issue by valvin1** [friendica/docker#95]
Hi,
I've just updated to 2019.12 and my cron container wasn't starting. 
I had to add `MYSQL_PORT` environment variable to make it works again. Else I had this error : 

```
Waiting for MySQL db initialization...
Port must be an integer greater than zero
[ERROR] Waited 300 seconds, no response
Waiting for MySQL db initialization...
Port must be an integer greater than zero
``` 

So I think it means default value is not set as before ?

**Comment by nupplaphil** [comment]
I'll check it :)

**Comment by nupplaphil** [comment]
Should be fixed with #96
